Owncloud iOs Client/AppDelegate.m

```diff
@@ -1721,14 +1710,16 @@ - (void)checkIfIsNecesaryShowPassCode {
             self.window.rootViewController = rootController;
             [self.window makeKeyAndVisible];
             
-            if (IS_IPHONE) {
-                
-                [rootController presentViewController:oc animated:YES completion:nil];
-                
-            } else {
+            if (!IS_IPHONE) {
                 oc.modalTransitionStyle = UIModalTransitionStyleCoverVertical;
                 oc.modalPresentationStyle = UIModalPresentationFormSheet;
-                [rootController presentViewController:oc animated:NO completion:nil];
+                
+            }
+            
+            if (_isPasscodeVisible == false){
```

Comment:
Is this condition needed? Isn't it already checked in the parent ```if``` before?

Also, don't you prefer syntax ```(!_isPasscodeVisible)```?